### PR TITLE
include installed features for isf

### LINKED ISSUE #19991: featureUtility does not pass all features from server.xml to repository resolver
**Describe the bug**  
After the fix in #19871, there are still issues where a featureUtility does not pass in the list of all of the features from the server.xml file to the repository resolver.  If a feature is already installed it ends up not passing it.  There may be difference decisions made on which features to install with a combination of features that are not made when only one feature is passed to the repository resolver.   As such all features need to be passed even if the features are installed.  I believe this should include if all features are already installed to make sure nothing else shows up as being needed.  If the list of feature returned from the resolveAsSet are all installed including all private and auto features, then there is truly nothing to install.

**Steps to Reproduce**  
There are easy ways to reproduce this issue.  

1. Download Jakarta EE 8 image from 21.0.0.12 or 22.0.0.1 and replace the repository.resolver bundle with one from a 22.0.0.2 build with #19871 resolved in it
2. Create a server with a server.xml file in it with the jakartaee-9.1 and federatedRegistry-1.0 feature in it.
3. Run featureUtility installServerFeatures serverName

**Expected behavior**  
The io.openliberty.wimcore.internal.ee-9.0.mf feature should be installed because with the combination of ee9 and federatedRegistry-1.0, it will choose the ee9 tolerated feature.

### PR DESCRIPTION
**Pull request by jjiwooLim**
Fixes #19991

### CODE REVIEW

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#libby-bot

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/Director.java [hunk 1]
```diff
@@ -507,21 +503,11 @@ public ServerPackageAsset resolveServerPackage(File archiveFile) throws InstallE
                 serverNames.add(sa.getServerName());
 
             }
-
```
Comment (jjiwooLim) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/Director.java:510]: This part (line 510-524)  is the change I made. All the other ones are due to indentation changes.

**Review by bensonlatibm — APPROVED** [review]
lgtm!

You might want to log when features are already installed, but it's not strictly necessary.

**Comment by wraschke** [comment]
We need to revert this because it causes a build break in LAOS continuous builds. See https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.workitem.viewWorkItem&id=289388.

**Pull request merged**